setup.py

```diff
@@ -10,12 +10,14 @@
     zip_safe=False,
     install_requires=[
         'celery',
+        'dogpile.cache',
         'flask',
         'flask-login',
         'flask-migrate',
         'flask-sqlalchemy',
         'operator-manifest',
         'psycopg2-binary',
+        'python-memcached',
```

Comment:
This is not a requirement since the default dogpile backend is null so it shouldn't be installed/required by default. That being said, it'd be nice to have it preinstalled in the workers container image. Perhaps [extras_require](https://setuptools.readthedocs.io/en/latest/userguide/dependency_management.html#optional-dependencies) might be the best option.

Thoughts @lcarva and @yashvardhannanavati?

Reply:
How would that work with the requirements.txt file? We'd also need another one of those?

Reply:
Yeah... it looks like pip-compile does not supports extras_require:
https://github.com/jazzband/pip-tools/blob/d2007dab6b0cd0c33860fcbf14da793fe34705da/piptools/scripts/compile.py#L379-L398

We could have it in `extras_require` and then install it via Ansible. I think it's too much hassle and we can move forward with just making it a dependency.

Reply:
Agreed. +1 on making it a dependency :+1: